config.yaml

```diff
@@ -233,6 +233,29 @@ docusaurus:
                               - Find Substring/: find-substring/
                               - Mul Arrays/: mul-arrays/
                               - Courses/: courses/
+      - Lab 8 - The Stack:
+          - Introduction:
+              path: chapters/stack/intro/
+              extra:
+                - media/
+              subsections:
+                - Reading/: reading/
+                - Guides:
+                    path: guides/
+                    subsections:
+                      - Stack Operations/: stack-operations/
+                      - Stack Addressing/: stack-addressing/
+                - Drills:
+                    path: drills/
+                    subsections:
+                      - Tasks:
+                          path: tasks/
+                          subsections:
+                            - Max/: max/
+                            - Reverse Array/: reverse-array/
+                            - Stack Addressing/: stack-addressing/
+                            - Local Var/: local-var/
+                            - Bonus - GCD/: gcd/
```

Comment:
After you split the reading material into subchapters, rearrange the guides and tasks and associate them with the newly created subchapters.